source/agora/test/Byzantine.d

```diff
@@ -232,6 +232,7 @@ private class ByzantineManager (bool addSpyValidator = false,
 
 private void waitForCount(size_t target_count, shared(size_t)* counter, string name)
 {
+    network.setTimeFor(Height(1));  // trigger consensus
```

Comment:
The `network` here seems to cause a unittest failure.

Reply:
I just pushed an update.